test/new-e2e/containers/ecs_test.go

```diff
@@ -24,6 +24,8 @@ import (
 )
 
 func TestAgentOnECS(t *testing.T) {
+	t.Parallel()
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Interesting, did not know that by default go can run multiple instances of a single test in //, that would break our executions  https://pkg.go.dev/testing#T.Parallel